### ISSUE TITLE
MM restriction system is working wrong

### ISSUE THREAD

**Issue by ziganshintimur** [ValveSoftware/Dota2-Gameplay#15183]
### Description

Current behavior system and MM restriction has following scenarios which working not correctly:
- 3-4+ reports by teammates for "fun, collusion, 3-4 people thinks differently, etc" in single game = low priority (this has to be reviewed by Overwatch...now its instant ban which means Overwatch review is not even taking place at all)
- 99 games in "normal" behavior is ok, 1 game outside "normal" = low priority (how can player be banned with 99+ games positive game history? there might be additional check, if player having perfect history why system gives immediate ban after 1 single game?)
- carry going wood for 50 mins joins zero fights - its okay hes carry, player on support hero stacking and trying to finish pipe/greaves/vlads/mech = low priority, since supports should not farm decided by both carry players
- Team lost all lanes and went afk fountain/wood waiting enemies to finish your throne? players who are still trying to win receive low priority 

Players also should not receive MM disabled for toxicity and have in-built mute chat/voice in game...this is just makes fun of the United States Constitution (The First Amendment provides that Congress make no law respecting an establishment of religion or prohibiting its free exercise. It protects freedom of speech, the press, assembly, and the right to petition the Government for a redress of grievances.)

So in all of the cases above - bans are immediate which leads to conclusion that excessive reports in 1 single game even with perfect history could lead to immediate ban (which is not how system should work for players who trying their best to win the game and having perfect history for many previous games)

Since Overwatch cases distributed between players once a day and taking the fact that people got immediate bans after 1 single game - it's makes sense that Overwatch cases review is just used to train automated ban system and not to actually ban/decline ban on exact case, and if so that's an abuse of players time which gives nothing in return

Please remove immediate bans from 1 single game for excessive reports count since system have zero evidence what happened (it clear that devs playing a lot with thresholds and sensitivity, but if player have 99+ game positive history and this thresholds triggers from 1 single game it just couldn't lead to immediate ban without additional review)

**Comment by Mentalsmelprob** [comment]
I subscribe to your every word. I have 41 praises for 15 games and my comrades gave me reports, having agreed in advance, the system gave me a ban for a WEEK. Extremely dissatisfied with Valve's new integrity system. It's just some kind of misunderstanding.
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/158267572/11a88662-dc78-4b05-b5ea-45088239b532)

**Comment by ziganshintimur** [comment]
Yes, current system not working correctly, too many people complains about it in tech support, reddit, github, thats why we should that problem for devs

**Comment by effect1111** [comment]
It has been almost half a year since the first complaints. They don't care
